### ISSUE TITLE
[2.4] Breakout Room Invites not send to everyone

### ISSUE THREAD

**Issue by GuiCrandall** [bigbluebutton/bigbluebutton#13322]
After Breaking out Rooms only some people get the invite to join the room. Other 2 professors in my institution reported the same error after we updated from BBB v.2.2.31 to BBB 2.4-rc-1.

I'm using MS Edge, but other professors and students are using other browsers for sure.

Appreciate your help and effort! 

https://user-images.githubusercontent.com/91569297/135162560-e2bdbb3f-442e-4a7a-b17a-623f8c0e5f5e.mp4

**Comment by mvasylenko** [comment]
I have expirienced the same problem. It seems that PR #13232 causes this behavior.

**Comment by GuiCrandall** [comment]
Yes, I saw the #13232 but that seemed to be a little different. Right?

**Comment by GuiCrandall** [comment]
Sorry I closed it accidentally...

**Comment by GuiCrandall** [comment]
Thank you. Tested and working...
